### PR TITLE
Integrated several loss components.

### PR DESCRIPTION
**Pull request by yilmazdoga**

### CODE REVIEW

File: odak/learn/perception/image_quality_losses.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                        Ground truth image.
+        peak_value    : float
+                        Peak value that given tensors could have.
+
+        Returns
+        -------
+        result        : torch.tensor
+                        Peak-signal-to-noise ratio.
+        """
+        mse = torch.mean((targets - predictions)**2)
```
Comment (kaanaksit) [inline odak/learn/perception/image_quality_losses.py:32]: `mse = torch.mean((targets - predictions) ** 2)`

File: odak/learn/perception/learned_perceptual_losses.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+        Parameters
+        ----------
+        device   : torch.device
+                    The device (CPU/GPU) on which the computations will be performed. Defaults to CPU.
+        """
+        super(CVVDP, self).__init__()
+        try:
+            import pycvvdp
+            self.cvvdp = pycvvdp.cvvdp(display_name='standard_4k', device=device)
```
Comment (kaanaksit) [inline odak/learn/perception/learned_perceptual_losses.py:19]: `self.cvvdp = pycvvdp.cvvdp(display_name = 'standard_4k', device = device)`
